ts/adaptors/HTMLAdaptor.ts

```diff
@@ -381,7 +381,11 @@ extends AbstractDOMAdaptor<N, T, D> implements MinHTMLAdaptor<N, T, D> {
      * @override
      */
     public setAttribute(node: N, name: string, value: string, ns: string = null) {
-        return (ns ? node.setAttributeNS(ns, name, value) : node.setAttribute(name, value));
+        if (!ns) {
+            return node.setAttribute(name, value);
+        }
+        name = ns.replace(/.*\//, '') + ':' + name.replace(/^.*:/, '');
+        return node.setAttributeNS(ns, name, value);
```

Comment:
I am a bit confused: when there is a namespace, we add it to the name, but then still provide the namespace explicitly?

Reply:
Yes, that is correct.  Because HTML5 removes namespaces, if you take `outerHTML` for an svg element in the browser, the `xlink:href` references will be listed as just `href` (no `xlink` regardless of whether the namespace.  This means that svg's that people save for use as a stand-alone will not load properly.  Doing it this way preserves the name correctly in `outerHTML` output.

Reply:
OK, thanks for the explanation.